packages/frontend/workspace/src/providers/sync/peer.ts

```diff
@@ -70,7 +71,8 @@ export class SyncPeer {
 
   constructor(
     private readonly rootDoc: Doc,
-    private readonly storage: Storage
+    private readonly storage: Storage,
+    private readonly priorityTarget = new PriorityTarget()
```

Comment:
better put it in queue

Reply:
agreed. maybe it will be easier to dedup the orchestration logic in line 228 ~ 241 etc